ext/libev/ev_select.c

```diff
@@ -193,26 +191,6 @@ select_poll (EV_P_ ev_tstamp timeout)
   int res;
   int fd_setsize;
 
-#ifdef _WIN32
```

Comment:
Once I thought NULL fds for select cause stuck, but https://github.com/tarcieri/cool.io/pull/47/files#diff-c9164d22260cba57b9d4a9beb08b42d0L258 says it will return EINVAL. I confirmed it with simple C program.
Now I think I misunderstood stuck on select() instead of stuck on Sleep().

Anyway this code still has a meaning as shortcut (skip select() and just sleep), but I choose code size.